jenkinsfile-runner-launcher

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ -n "$DEBUG" ] ; then
+  export JAVA_OPTS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005 $JAVA_OPTS"
```

Comment:
Would it be possible to specify this in the same call than the command so that we don't change the value of the environment variable?

Reply:
The command is generated by appassembler, so the only way to pass this arguments is using the environment variable:

```
exec "$JAVACMD" $JAVA_OPTS  \
  -classpath "$CLASSPATH" \
  -Dapp.name="jenkinsfile-runner" \
  -Dapp.pid="$$" \
  -Dapp.repo="$REPO" \
  -Dapp.home="$BASEDIR" \
  -Dbasedir="$BASEDIR" \
  io.jenkins.jenkinsfile.runner.bootstrap.Bootstrap \
  "$@"
```

Reply:
Yes, but would something like "JAVA_OPTS=xxx /app/bin/jenkinsfile-runner" work?

Reply:
I don't think so. That's changing the JAVA_OPTS equally. Or might I be missing something on your proposal?

Reply:
I meant the JAVA_OPTS to be changed permanently, but the export is only inside the script so you can forget about my comment.